src/ifcconvert/ColladaSerializer.h

```diff
@@ -69,11 +69,13 @@ class ColladaSerializer : public GeometrySerializer
 			{}
             void addFloatSource(const std::string& mesh_id, const std::string& suffix,
                 const std::vector<real_t>& floats, const char* coords = "XYZ");
-            void write(const std::string &mesh_id, const std::string& default_material_name,
+            /// @todo pass simply DeferredObject?
```

Comment:
> @todo pass simply DeferredObject?

Or forget about this DeferredObject and (a) copy constructor for IfcGeom::Element, or (b) `std::unique_ptr<>&& Iterator::get()`?